filters/resources.txt

```diff
@@ -2310,3 +2310,34 @@ damoh-defuser.js application/javascript
     var observer = new MutationObserver(cleanVideoAsync);
     observer.observe(document.documentElement, { childList: true, subtree: true });
 })();
+
+
+twitch-videoad.js application/javascript
+(function() {
+	var ourMediaPlayer = undefined;
+	Object.defineProperty(window, 'MediaPlayer', {
+		set: function(newMediaPlayer) {
+			if (typeof(ourMediaPlayer) !== 'undefined')
+				return;
+			var oldLoad = newMediaPlayer.MediaPlayer.prototype.load;
+			newMediaPlayer.MediaPlayer.prototype.load = function(e) {
+				try {
+					if (e.startsWith('https://usher.ttvnw.net/api/channel/hls/')) {
+						var url = new URL(e);
+						url.searchParams.delete('baking_bread');
+						url.searchParams.delete('baking_brownies');
+						url.searchParams.delete('baking_brownies_timeout');
+						e = url.href;
```

Comment:
From my testing this is indeed enough to bypass the preroll video ads. Not sure about ads pushed by the streamer during playback tho. But those can be dealt with later if needed.